kube-client/src/client/auth/mod.rs

```diff
@@ -500,6 +508,25 @@ fn auth_exec(auth: &ExecConfig) -> Result<ExecCredential, Error> {
         cmd.envs(envs);
     }
 
+    let interactive = auth.interactive_mode != Some(ExecInteractiveMode::Never);
+    if interactive {
+        cmd.stdin(std::process::Stdio::inherit());
+    } else {
+        cmd.stdin(std::process::Stdio::piped());
+    }
```

Comment:
Double checked this with [std::process::Command::stdin](https://doc.rust-lang.org/std/process/struct.Command.html#method.stdin) docs and this should indeed be a non-breaking addition.